azure-pipelines.yml

```diff
@@ -33,6 +43,12 @@ steps:
   - script: |
       npm run test
     displayName: "test"
+
+  - script: |
+      npm run test-e2e:open
+      npm run test-e2e:run
```

Comment:
For the integration phase `test-e2e:run` would be sufficient. There is no need running `test-e2e:open` as well. Or am I being mistaken?

Reply:
I thought that I need to Open a cypress "session" before running the tests. Should I remove the tests-e2e:open?

Reply:
Yeah, `test-e2e:run` will handle it for you - you can even test it locally ✌️